udata/templates/reuse/card.html

```diff
@@ -1,34 +1,32 @@
 {% cache cache_duration, 'reuse-card', reuse.id|string, g.lang_code %}
-<div class="card reuse-card">
+{% set features = features or [] %}
+<div class="card reuse-card {% if 'preview' in features %}card--with-cover{% endif %}">
+    {% if 'preview' in features %}
```

Comment:
What is `features.preview`?

Reply:
Maybe the label is wrong — that's what makes the card to display the reuse cover (in the Community Reuses examples) vs. the Reuse in an org listing.

I'm happy to stick with one design only to mitigate complexity.

Reply:
Ok, why not!

Reply:
(why not for 2 designs, seems to make sense)